### PR TITLE
Recover email validation request

### PR DESCRIPTION
**Pull request by maxlath**
It use to be possible to request a new email with a token to validation the user email address; that was lost when the settings got rewritten in Svelte, and this PR brings that back

### CODE REVIEW

**Review by jum-s — CHANGES_REQUESTED** [review]
this is graphically getting a lot of user's attention, but its neither an error nor even a problem, only optional button if one havent received an email.

Instead i would integrate it the same way as other points in the settings
Title: "unverified email"
Message: in warning color with the current icon
Action button: like any other

on the blocking points:
-    align text on the left (as every text of the whole settings)
-    keep action button to a regular color, this black is not very spread (i would wait to refactor modals with svelte to introduce new action colors)
-    no background color for this section (same as any other section in settings)

**Comment by maxlath** [comment]
@jum-s style harmonization done in 8f548eb

File: app/modules/settings/components/email_validation.svelte [hunk 1]
```diff
@@ -15,54 +15,17 @@
...
+  <p>
+    {@html icon('check')}
+    {@html I18n('new_confirmation_email')}
+  </p>
+{:else}
+  <p>
+    {@html icon('warning')}
+    {@html I18n('email_wasnt_verified')}
+  </p>
+  <button class="button light-blue radius" on:click={emailConfirmationRequest}>
```
Comment (jum-s) [inline app/modules/settings/components/email_validation.svelte:28]:
```suggestion
  <button class="light-blue-button radius" on:click={emailConfirmationRequest}>
```

Comment (maxlath) [reply app/modules/settings/components/email_validation.svelte]: done in 59c863a

**Review by jum-s — APPROVED** [review]

**Pull request merged**